src/modal/modal.component.ts

```diff
@@ -100,20 +92,9 @@ export class ModalDemo {
 				<ng-content></ng-content>
 			</div>
 		</ibm-overlay>
-	`,
-	animations: [
```

Comment:
Why are we removing the animations from modal? The react implementation still has it

Reply:
This doesn't remove the animations, since those are in CSS ... The Angular animations never actually did anything useful

Reply:
I played around with the changes and it seems like the animations were removed

Reply:
Oh weird

Reply:
Should we be dynamically setting some classes on the modal that we don't at the moment?

Reply:
it's possible ... I'll have to dig into it later

Reply:
Should it be kept for now?

Reply:
This isn't going into a release so it's not super critical ... We'll want to make sure we're using the official carbon timings, so I don't think we'll be adding back the Angular animations